setup.py

```diff
@@ -32,7 +32,7 @@
     # Versions should comply with PEP440.  For a discussion on single-sourcing
     # the version across setup.py and the project code, see
     # https://packaging.python.org/en/latest/single_source_version.html
-    version='0.1',
+    version='0.5',
```

Comment:
Definitely not 1.0! But I want to make sure @horsburgh says he's comfortable with that big of a jump, to 0.5, for @sreeder's major additions.

I don't think there's any value in sending/preserving 0.1 (the 0.1-alpha tagged pre-release) to live pypi, since the API will change so much after 0.1. FYI, we went ahead with 0.1-alpha on the anaconda channel just to be able to make progress in setting up the channel.